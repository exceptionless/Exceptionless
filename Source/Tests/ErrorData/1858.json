{
  "Id": "80000000818f3b0fe0e9b146",
  "OccurrenceDate": "2014-03-05T21:39:10.72-06:00",
  "Tags": [
    "1.8.0.1930",
    "v1.8"
  ],
  "ExceptionlessClientInfo": {
    "Version": "1.3.1858 5533107c17",
    "InstallDate": "2014-03-02T15:47:36-06:00",
    "StartCount": 0,
    "SubmitCount": 5,
    "Platform": ".NET",
    "SubmissionMethod": "Manual"
  },
  "Modules": [
    {
      "ModuleId": 1,
      "Name": "Example.Utils.v12.2",
      "Version": "12.2.1.0",
      "IsEntry": false,
      "CreatedDate": "2014-02-28T07:09:32.903+00:00",
      "ModifiedDate": "2013-09-01T20:52:10+00:00",
      "ExtendedData": {
        "PublicKeyToken": "b88d1754d700e49a",
        "FileVersion": "12.2.1.13242"
      }
    }
  ],
  "ExtendedData": {},
  "StackTrace": []
}