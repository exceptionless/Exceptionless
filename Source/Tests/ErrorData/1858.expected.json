{
  "ReferenceId": "80000000818f3b0fe0e9b146",
  "Date": "2014-03-05T21:39:10.72-06:00",
  "Tags": [
    "1.8.0.1930",
    "v1.8"
  ],
  "Type": "error",
  "@error": {
    "Modules": [
      {
        "ModuleId": 1,
        "Name": "Example.Utils.v12.2",
        "Version": "12.2.1.0",
        "IsEntry": false,
        "CreatedDate": "2014-02-28T07:09:32.903+00:00",
        "ModifiedDate": "2013-09-01T20:52:10+00:00",
        "Data": {
          "PublicKeyToken": "b88d1754d700e49a",
          "FileVersion": "12.2.1.13242"
        }
      }
    ]
  }
}